pandas/tests/frame/test_constructors.py

```diff
@@ -753,7 +754,7 @@ def test_constructor_extension_scalar_data(self, data, dtype):
         assert df["a"].dtype == dtype
         assert df["b"].dtype == dtype
 
-        arr = pd.array([data] * 2, dtype=dtype)
+        arr = pd_array([data] * 2, dtype=dtype)
```

Comment:
What's the advantage of using such an alias instead of simply using `pd.array` ?

Reply:
that in this file there is `import array`, so having `pd.array` as well would fail the check

Reply:
Yes, but I meant from a developer (code writer / reader) perspective ;)  
I understand that the check has difficulties with it, but then I would say to either fix the check or ignore "array" in the check if it's not easy to fix (IMO array in tests should always be used as `pd.array`, so it might worth a special case in the checking code)

Reply:
sure, I can add it to `EXCLUDE` (along with `eval` and `np`)

Reply:
Thnks!